sdk/template-dpg/Azure.ServiceTemplate.Template/Azure.ServiceTemplate.Template.sln

```diff
@@ -0,0 +1,53 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29709.97
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core.TestFramework", "..\..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj", "{ECC730C1-4AEA-420C-916A-66B19B79E4DC}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Azure.ServiceTemplate.Template", "src\Azure.ServiceTemplate.Template.csproj", "{AF06AB51-3B39-4905-B7EA-CD7563DE5770}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ServiceTemplate.Template.Tests", "tests\Azure.ServiceTemplate.Template.Tests.csproj", "{ADB00BD6-8FEE-4176-BE15-4F530F5C8F29}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Azure.Core", "..\..\core\Azure.Core\src\Azure.Core.csproj", "{85677AD3-C214-42FA-AE6E-49B956CAC8DC}"
```

Comment:
We should use a package reference to Azure.Core and Azure.Core.Experimental.

Reply:
updated